src/mod/applications/mod_callcenter/mod_callcenter.c

```diff
@@ -4356,4 +4518,4 @@ SWITCH_MODULE_SHUTDOWN_FUNCTION(mod_callcenter_shutdown)
  * End:
  * For VIM:
  * vim:set softtabstop=4 shiftwidth=4 tabstop=4 noet
- */
+ */
```

Comment:
Missing a new line at the end of the file... (It just annoying).

Reply:
done